readme.md

```diff
@@ -53,98 +50,64 @@ Broken into a variety of functional areas, the eligibility criteria is a checkli
+- When applicable, project must use reproducible build settings in its toolchain.
+- The project uses [Source Link](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/sourcelink).
+- The project uses either embedded PDBs or publish symbol packages to NuGet (if applicable).
+- The project code signs their artifacts as appropriate.
 
 **Licenses and Copyright**
 
--The code is offered under a standard, permissive open source license which has been approved by the .NET Foundation.
-
--Libraries that are mandatory dependencies of the project are also offered under a standard, permissive open source license which has been approved by the .NET Foundation (exceptions include a dependency on .NET Framework).
-
--Committers are bound by a Contributor License Agreement.
-
--The copyright ownership of everything that the project produces is clearly defined and documented.
+- The code is offered under a standard, permissive open source license which has been approved by the .NET Foundation.
+- Libraries that are mandatory dependencies of the project are also offered under a standard, permissive open source license which has been approved by the .NET Foundation (exceptions include a dependency on .NET Framework).
+- Committers are bound by a Contributor License Agreement.
+- The copyright ownership of everything that the project produces is clearly defined and documented.
 
 **Quality**
 
--The project is open and honest about the quality of its code.
-
--The project has a public issue tracker where the status of any defect can be easily obtained.
-
--The project puts a very high priority on producing secure software. 
-
--The project has a published Security Policy.
+- The project is open and honest about the quality of its code.
+- The project has a public issue tracker where the status of any defect can be easily obtained.
+- The project puts a very high priority on producing secure software. 
+- The project has a published Security Policy.
 
 **Community**
 
--The project has a home page which provides high level information about its status and purpose.
-
--The project welcomes contributions from anyone and embraces diversity in all forms.
-
--The project strives to be a meritocracy where more rights and responsibilities are granted to contributors who add value to the project.
-
--The project has a public communication channel where community members can engage with maintainers.
-
--The project adheres to, and enforces, the .NET Foundation Code of Conduct.
-
--The project has a publicly available location where members can review documentation.
+- The project has a home page which provides high level information about its status and purpose.
+- The project welcomes contributions from anyone and embraces diversity in all forms.
+- The project strives to be a meritocracy where more rights and responsibilities are granted to contributors who add value to the project.
```

Comment:
Strongly feel we should not embrace the word "meritocracy" here. It was [coined as satire](https://kottke.org/17/03/the-satirical-origins-of-the-meritocracy) and does not mean what we intend it to mean here to many people. In fact, it can be a divisive word: https://readwrite.com/2014/01/24/github-meritocracy-rug/

Reply:
I was not aware of the origin of the word and I have never seen it used in a divisive manner. In fact, I see it regularly used in open source communities and it seems that the majority of people are familiar with the more traditional definition of the word - ie. https://www.merriam-webster.com/dictionary/meritocracy or https://en.wikipedia.org/wiki/Meritocracy

Reply:
@sbwalker the wikipedia article you referenced has a whole section of Criticism on the term meritocracy which begins with...

> The term "meritocracy" was originally intended as a negative concept.

And continues to highlight some of the issues with the term. The term meritocracy is often used to push out members of marginalized groups. There's [a discussion in the Geek Feminism Wiki ](https://geekfeminism.wikia.org/wiki/Meritocracy) that cites several criticisms of the concept.